app/src/foo.c

```diff
@@ -0,0 +1,12 @@
+/**
+ * @author Teslabs Engineering S.L.
```

Comment:
Author and copyright format differ from built-in samples. Why? Again, I think consistency matters here.

Reply:
I just copied from some apps I have. This format makes Doxygen output nicer, but I agree consistency is better.

Reply:
adjusted